src/unistd.rs

```diff
@@ -2524,6 +2527,51 @@ mod setres {
     }
 }
 
+#[cfg(any(target_os = "android", target_os = "linux", target_os = "openbsd"))]
```

Comment:
This will fail to compile for OpenBSD.  You need to open a PR with https://github.com/rust-lang/libc first.